frontend/src/metabase/common/components/SaveQuestionForm/util.ts

```diff
@@ -92,6 +93,21 @@ const getName = (question: Question, originalQuestion: Question | null) => {
   }
 
   // Ad-hoc query
+  // For metrics, we need to generate a name without breakouts
+  // since metrics are reusable aggregations that can be used with any grouping
+  if (question.type() === "metric") {
+    let query = question.query();
+
+    // Remove all breakouts to get a clean metric name
+    const breakouts = Lib.breakouts(query, -1);
+    breakouts.forEach((breakoutClause: any) => {
+      query = Lib.removeClause(query, -1, breakoutClause);
+    });
```

Comment:
We could add this "remove all breakouts" to the lib easily, which would be faster/simpler. But this is a rare operation so it's probably fine.